ui/src/main/java/com/wireguard/android/activity/TvMainActivity.kt

```diff
@@ -321,6 +322,16 @@ class TvMainActivity : AppCompatActivity() {
         binding.tunnelList.forEach { viewItem ->
             val listItem = DataBindingUtil.findBinding<TvTunnelListItemBinding>(viewItem)
                     ?: return@forEach
+            if (binding.tunnelList.size>0){
```

Comment:
Formatting!!!

Reply:
And what do we do if the `tunnelList` is 0? Does the NRG app get opened nevertheless?

Reply:
NRG app is opened just in case there are tunnels in tunnel List.